### PR TITLE
Add criticalup.toml docs

### PR DESCRIPTION
**Pull request by Hoverbear**
Some more simple improvements to the docs.

### CODE REVIEW

**Comment by pietroalbini** [comment]
There are merge conflicts :(

**Comment by pietroalbini** [comment]
Rebased, and updated the download URLs.

File: docs/src/criticalup_toml.rst [hunk 1]
```diff
@@ -4,4 +4,109 @@
...
+    # ...
+
+``packages``
+^^^^^^^^^^^^
+
+A set of package names as listed in the release page, for example the `beta-stable-24.05.0
+<https://releases.ferrocene.dev/ferrocene/files/stable-24.05.0/index.html>`_
+release.
+
+Each supported Ferrocene target lists required packages in the on it's page
```
Comment (jonathanpallant) [inline docs/src/criticalup_toml.rst:97]:
```suggestion
Each supported Ferrocene target lists required packages in the on its page
```

File: docs/src/install.rst [hunk 2]
```diff
@@ -47,6 +48,6 @@ From a terminal run:
 
 .. code-block::
 
-   powershell -c "irm https://github.com/ferrocene/criticalup/releases/download/criticalup-cli-v0.0.0/criticalup-cli-installer.ps1 | iex"
+   powershell -c "irm https://github.com/ferrocene/criticalup/releases/latest/download/criticalup-installer.ps1 | iex"
 
-CriticalUp will install into ``{FOLDERID_RoamingAppData}``, usually ``%appdata%\\criticalup``.
\ No newline at end of file
+CriticalUp will install toolchains into ``{FOLDERID_RoamingAppData}``, usually ``%appdata%\\criticalup``.
```
Comment (jonathanpallant) [inline docs/src/install.rst:53]:
```suggestion
CriticalUp will install toolchains into ``{FOLDERID_RoamingAppData}``, usually ``%appdata%\criticalup``.
```


I don't think this backslash needs escaping.

**Review by jonathanpallant — DISMISSED** [review]

File: docs/src/criticalup_toml.rst [hunk 3]
```diff
@@ -4,4 +4,109 @@
...
+.. code-block::
+
+    [products.ferrocene]
+    release = "stable-24.05.0"
+    # ...
+
+``packages``
+^^^^^^^^^^^^
+
+A set of package names as listed in the release page, for example the `beta-stable-24.05.0
```
Comment (Urhengulas) [inline docs/src/criticalup_toml.rst:93]: Is `beta-stable-24.05.0` correct?

Comment (pietroalbini) [reply docs/src/criticalup_toml.rst:93]: That's a typo, will fix.

**Review by jonathanpallant — APPROVED** [review]

**Comment by pietroalbini** [comment]
bors merge

**Pull request merged**